### DISCUSSION TITLE
Why is the cord connected to other handles?

### DISCUSSION THREAD

**Discussion by spy6sky** [bcakmakoglu/vue-flow#1709] [Q&A]
![企业微信截图_1733391116716](https://github.com/user-attachments/assets/d5c961a1-e0f7-4af4-9d64-941b7a2fd7be)
![企业微信截图_17333911338628](https://github.com/user-attachments/assets/a498948d-8737-42a8-aac0-6dc804d6a6c1)

As shown in the figure, the position where the line should be connected has changed. This situation is difficult to reproduce, but it does exist. I used dagrejs to set it up. I can't determine which direction the problem occurred now. It is a problem with the custom node, or a problem that occurred when using dagrejs to draw.
ps: I did it with reference to https://vueflow.dev/examples/layout/animated.html. Mine has no custom edges.

**Comment by bcakmakoglu** [comment]
Can you provide a reproduction of this issue?

[You can find a template here.](https://codesandbox.io/p/sandbox/vue-flow-basic-gfgro4)
